ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/PaywallFont.kt

```diff
@@ -33,9 +33,9 @@ sealed class PaywallFont : Parcelable {
         val fontWeight: FontWeight = FontWeight.Normal,
         /**
          * The style of the font, normal or italic. The system uses this to match a font to a font request.
+         * We use int instead of [FontStyle] because [FontStyle] is not compatible with Java.
          */
-        @TypeParceler<FontStyle, FontStyleParceler>()
-        val fontStyle: FontStyle = FontStyle.Normal,
+        val fontStyle: Int = FontStyle.Normal.value,
```

Comment:
This is another breaking change in kotlin... Basically I'm doing this since it made it impossible to create instances of these classes in Java, since `FontStyle` is not available in Java.